BootloaderCommonPkg/Include/Guid/LoaderPlatformInfoGuid.h

```diff
@@ -45,12 +45,6 @@ extern EFI_GUID gLoaderPlatformInfoGuid;
 //Secure debug (0 - disabled, 1 - enabled)
 #define        HWSTATE_SECURE_DEBUG              BIT3
 
-//Tpm Type dTPM 2.0 (0 - Not Availble, 1 - detected)
-#define        HWSTATE_DTPM_20                   BIT4
-
-//Tpm Type TPM 2.0 PTT (0 - Not Availble, 1 - detected)
-#define        HWSTATE_TPM_PTT                   BIT5
-
```

Comment:
Better still use HWSTATE instead adding a new field? we don't need change code, e.g.
#define  HWSTATE_TPM_MASK          (BIT4 | BIT5)
#define  HWSTATE_TPM_SHIFT           4
#define  TPM_TYPE_NOT_KNOWN     (0x0 << HWSTATE_TPM_SHIFT)
#define  TPM_TYPE_NONE                 (0x1 << HWSTATE_TPM_SHIFT)
...

Reply:
I would think it makes sense to use separate new field in this case.

Reply:
HwState is maintained bit wise for each feature. As TpmType is additional info, it think it would be good to have a separate field.

Reply:
If we add new field, do need update structure version? and consumer need check the version to know if new field is there before consuming TPM information.

Reply:
I would update the structure version to "2" and check before we consume it.

Reply:
It is ok for me if update version and check it.